src/components/Navbar/Navbar.js

```diff
@@ -104,7 +104,11 @@ class Navbar extends Component {
               <Switch>
                 <Route exact path="/Profile" component={Profile} />
                 <Route exact path="/userDashboard" component={UserDashboard} />
-                <Route exact path="/vendorDashboard" component={VendorDashboard} />
+                <Route
```

Comment:
Keep this in single line. Breaking up each route in separate line simply makes the code long and difficult to read in future.

Reply:
Changes made , it was the editors fault which reset it to this